pipeline/astro_cal.py

```diff
@@ -234,7 +238,7 @@ def _run_scamp( self, ds, prov, session=None ):
                 _logger.info( f"Failed to solve for WCS with maxmag {maxmag}, trying the next one." )
 
         if not success:
-            raise RuntimeError( "_run_scamp failed to find a match." )
+            raise RuntimeError( f"_run_scamp failed to find a match. Exceptions that were raised: {exceptions}" )
```

Comment:
This could potentially be a very long error message....

Reply:
Better to have a long error than nothing.